README.md

```diff
@@ -2,20 +2,24 @@
 
 ![a list of pull requests in the GitHub Pull Requests](https://raw.githubusercontent.com/RoadieHQ/backstage-plugin-github-pull-requests/main/docs/list-of-pull-requests-and-stats-tab-view.png)
 
+
```

Comment:
Do this step next: https://github.blog/2017-11-08-archiving-repositories/

So we'll get a nice yellow banner on the top as well

Reply:
Thank you for this, cool thing to have :)